composer.json

```diff
@@ -7,6 +7,34 @@
         "coding standard"
     ],
     "require": {
-        "squizlabs/php_codesniffer": "^2.7"
+        "squizlabs/php_codesniffer": "3.0.1"
```

Comment:
Hm.... even if the bug fix release change behaviour of some sniffs? Do we want to behave different with different version of dependencies? I think it will be better if we release always new version with updated dependencies when needed - so always running tests with lowest/latest dependencies gives us the same results.